docker/static/Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN opam switch create 4.07 ocaml-base-compiler.4.07.1 && \
   opam switch 4.07
 
 # Stanc3 also needs
-RUN opam install -y menhir core_kernel ppx_deriving
+COPY ./scripts/install_build_deps.sh ./
+RUN bash -x install_build_deps.sh
```

Comment:
This caught me - I had added the new dependency on fmt to the build script but not to this Dockerfile, so I figured it was best to keep the authoritative list in one place.

Reply:
Good idea!